src/main/java/com/amplitude/AmplitudeLog.java

```diff
@@ -0,0 +1,38 @@
+package com.amplitude;
+
+public class AmplitudeLog {
+
+    private static LogMode LOG_MODE = LogMode.DEBUG;
+
+    public static void setLogMode(LogMode logMode) {
+        AmplitudeLog.LOG_MODE = logMode;
+    }
+
+    public static void log(String tag, String message) {
+        log(tag, message, LogMode.DEBUG);
+    }
+
+    public static void log(String tag, String message, LogMode messageMode) {
```

Comment:
You can have other functions to call this with LogMode set.
`debug(...)`
`error(...)`
`warn(...)`